trending_data/src/main/java/com/google/sps/servlets/TweetsGetterServlet.java

```diff
@@ -0,0 +1,46 @@
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import twitter4j.Query;
+import twitter4j.QueryResult;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import twitter4j.TwitterFactory;
+
+@WebServlet("/get-tweets")
```

Comment:
Could you add a comment explaining what the servlet does? I.e. "Given a search query, finds 5 recent/popular matching tweets."